homeassistant/helpers/device_registry.py

```diff
@@ -247,7 +247,10 @@ def _async_remove_device(self, device_id):
                     sw_version=device['sw_version'],
                     id=device['id'],
                     # Introduced in 0.79
-                    hub_device_id=device.get('hub_device_id'),
+                    # renamed in 0.95
+                    via_device_id=(
+                        device.get('via_device_id')
```

Comment:
This automatically upgrades the existing device registry on loading. Alternately we could do an update script instead, what's better here?

Reply:
This is a lot smaller than adding a script 👍 . Just make renamed -> Renamed